tests/puppeteer/pages/BO/shopParameters/productSettings/index.js

```diff
@@ -77,4 +78,15 @@ module.exports = class productSettings extends BOBasePage {
     await this.clickAndWaitForNavigation(this.saveProductGeneralFormButton);
     return this.getTextContent(this.alertSuccessBloc);
   }
+
+  /**
+   * Change default activation status
+   * @param toEnable
+   * @returns {Promise<string|*>}
+   */
+  async setDefaultActivationStatus(toEnable = true) {
+    await this.waitForSelectorAndClick(this.switchDefaultActivationStatusLabel.replace('%TOGGLE', toEnable ? 1 : 0));
+    await this.clickAndWaitForNavigation(this.saveProductGeneralFormButton);
+    return this.getTextContent(this.alertSuccessBloc);
```

Comment:
```suggestion
    return this.getTextContent(this.alertSuccessBlock);
```

Reply:
The selector name is used in many other files, maybe I should create another PR to fix the name in all files. 
Wdyt?

Reply:
:ok_hand: for an another PR